addon/eventer.js

```diff
@@ -1,69 +0,0 @@
-import jQuery from 'jquery';
```

Comment:
This file seems to not be used at all!?

Reply:
if no one imports it, i'd say yep -- looks like an initial or refactor attempt at the event dispatcher override...